sklearn/tests/test_docstring_parameters.py

```diff
@@ -229,7 +228,7 @@ def test_fit_docstring_attributes(name, Estimator):
         est.set_params(n_components="auto")
 
     if Estimator.__name__ == "QuantileRegressor":
```

Comment:
Same remark you can remove the `if` block here too

Reply:
Removed.